pkgs/development/python-modules/hawkmoth/default.nix

```diff
@@ -0,0 +1,39 @@
+{
+  lib,
+  buildPythonPackage,
+  pythonOlder,
+  fetchPypi,
+  hatchling,
+  sphinx,
+  libclang,
+}:
+
+buildPythonPackage rec {
+  pname = "hawkmoth";
+  version = "0.19.0";
+  pyproject = true;
+
+  disabled = pythonOlder "3.9";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-NxjSUg/ZznuAKItzbc/sInZGbQxJLXy0HLyCLxLK8yo=";
+  };
```

Comment:
fetchFromGitHub is preferred unless there's a compelling reason to use Pypi, in which case add a comment

Reply:
Switched to github.